misc/config_tools/schema/checks/rdt_support.xsd

```diff
@@ -7,15 +7,15 @@
                    then (//CDP_ENABLED = 'n' and //RDT_ENABLED = 'y')
                    else true()">
     <xs:annotation>
-      <xs:documentation>vCAT can be enabled only when RDT_ENABLED is 'y' and CDP_ENABLED is 'n'</xs:documentation>
+      <xs:documentation>Hypervisor Virtual Cache Allocation Technology may be enabled only when hypervisor Intel Resource Director Technology is enabled and hypervisor Code and Data Prioritization is disabled.</xs:documentation>
```

Comment:
In the UI, the checkbox is named Cache Allocation Technology. Intel Resource Director Technology is just a heading. But it is being discussed in JIRA 9058. This comment applies to all messages that mention enabling RDT below.